content/_index.md

```diff
@@ -7,7 +7,7 @@ outputs = ["Reveal"]
 
 # Parallel Forecast
 
-Working to build better systems for generating common knowledge.
+We’re a team of alumni from Oxford and Google building more reliable methods for forecasting AI Progress.
```

Comment:
I don't like leading with "team of alumni" - whatever comes first is center stage, and this feels like an 'appeal to resume'.

I'm unsure but could see the value of explicitly saying we're working on AI Progress. 

"Working to build better ways to forecast the future of tech. We're starting with AI - contact us for more info". 

Github isn't the best place for iterating on text, so I'd prefer to continue this in a google doc or whats app.